src/renderer/src/routes/(main)/app-store/[slug]/+page.svelte

```diff
@@ -55,13 +61,22 @@
 	const handleError = (error: unknown, versionEntity?: Entity<AppVersionEntry>) => {
 		console.error(error);
```

Comment:
The error is already always logged to the console but now we change the text shown in the dialog to be end-user-readable and potentially actionable.